drivers/ieee802154/Kconfig

```diff
@@ -10,6 +10,20 @@ menuconfig IEEE802154
 	bool "IEEE 802.15.4 drivers options"
 	default y if NET_L2_IEEE802154 || NET_L2_OPENTHREAD
 
+config IEEE802154_NET_IF_NO_AUTO_START
+	bool "Disable 802.15.4 interface auto-start"
+	depends on NET_L2_OPENTHREAD || NET_L2_IEEE802154
```

Comment:
you don't need that depends, since the menu has the same dependencies.

Reply:
done